common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerRequest.java

```diff
@@ -94,21 +94,18 @@ private static final class SerializedNames {
      * Scopes for the request. This is expected to be of the format
      * "scope 1 scope2 scope3" with space as a delimiter
      */
-    @NonNull
     @SerializedName(SerializedNames.SCOPE)
     private String mScope;
 
     /**
      * The redirect uri for the request.
      */
-    @NonNull
     @SerializedName(SerializedNames.REDIRECT)
     private String mRedirect;
 
     /**
      * The client id of the application.
      */
-    @NonNull
```

Comment:
Why removing these NonNull annotations?

Reply:
BrokerRequest can then be used for operations other than AT/ATS which mandate this and can represent generic broker request blob (serializable) to broker. On Broker side AT/ATS operations do validate parameters they receive.

Let me know if you have different opinion, I can consider creating separate request object for this request.